inc/class-cachify.php

```diff
@@ -1334,6 +1334,18 @@ public static function set_cache( $data ) {
 
 		/* Save? */
 		if ( $should_cache ) {
+			/**
+			 * Filters the buffered data itself
+			 *
+			 * @since 2.3.2
+			 *
+			 * @param string $data          The actual data.
+			 * @param object $method        Instance of the selected caching method.
+			 * @param string $cache_hash    The cache hash.
+			 * @param int    $cache_expires Cache validity period.
+			 */
+			$data = apply_filters( 'cachify_buffer', $data, self::$method, self::_cache_hash(), self::_cache_expires() );
```

Comment:
The name `cachify_buffer` is not quite self-explaining (there is no such thing like a "buffer" from an outside perspective). I'd prefer something like `cachify_pre_store` or `cachify_modify_output`.

Reply:
Good point! Changed it to `cachify_modify_output`.